File: 프로그래머스/lv1/12901. 2016년/2016년.cs
public class Solution
{
    public string solution(int a, int b)
    {
        int[] month = new int[] { 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
        int answer = 4;
        for (int i = 0; i < a - 1; i++)
        {
            answer += month[i];
        }
        answer = (answer + b) % 7;
        return new string[] { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" }[answer];
    }
}

File: 프로그래머스/lv1/12901. 2016년/2016년.py
# 1월 1일은 금요일
def solution(a, b):
    return ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"][(sum([31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31][:a-1]) + b + 4) % 7]